app/defines/interrupt_handler.h

```diff
@@ -10,7 +11,10 @@
 
 unsigned int COUNT_MACHINE_TIMER_INTRPT[1] = {0};
 unsigned int funct7, funct3;
-unsigned int rs1, rs2, rd;
+unsigned int rs1_addr, rs2_addr, rd_addr;
```

Comment:
i think its a "pointer" not an "address"

i think the terminology address should be for memory.
This is more like a rs1_rf_ptr (reg_soruce_1_register_file_pointer)